src/app/pages/HomePage/Graph/Graph/index.tsx

```diff
@@ -0,0 +1,641 @@
+        fill="white"
+      />
+      <g style={{ pointerEvents: selectedLayer === Layer.Emerald ? 'auto' : 'none' }}>
+        <Link to={RouteUtils.getLatestTransactionsRoute(Layer.Emerald)}>
+          <circle
+            cx="234.552"
+            cy="36.6031"
+            r="11.8332"
+            transform="rotate(-90 234.552 36.6031)"
+            fill="#01F1E3"
+          />
+          <circle
+            cx="234.482"
+            cy="36.6743"
+            r="8.02878"
+            transform="rotate(-90 234.482 36.6743)"
+            fill="#191932"
+          />
+          <text x="231.5" y="38" fill="#fff" fontSize="4px">
+            {t('home.txn')}
+          </text>
+        </Link>
+        <Link to={RouteUtils.getLatestBlocksRoute(Layer.Emerald)}>
+          <circle
+            cx="155.943"
+            cy="37.8709"
+            r="11.8332"
+            transform="rotate(-90 155.943 37.8709)"
+            fill="#01F1E3"
+          />
+          <circle
+            cx="155.873"
+            cy="37.9424"
+            r="8.02878"
+            transform="rotate(-90 155.873 37.9424)"
+            fill="#191932"
+          />
+          <text x="150.5" y="39" fill="#fff" fontSize="4px">
+            {t('home.blocks')}
+          </text>
+        </Link>
+      </g>
+
+      <g style={{ pointerEvents: selectedLayer === Layer.Sapphire ? 'auto' : 'none' }}>
+        <Link to={RouteUtils.getLatestTransactionsRoute(Layer.Sapphire)}>
+          <circle
+            cx="155.833"
+            cy="295.833"
+            r="11.8332"
+            transform="rotate(-90 155.833 295.833)"
+            fill="#01F1E3"
+          />
+          <circle
+            cx="155.763"
+            cy="295.904"
+            r="8.02878"
+            transform="rotate(-90 155.763 295.904)"
+            fill="#191932"
+          />
+          <text x="152.5" y="297" fill="#fff" fontSize="4px">
+            {t('home.txn')}
+          </text>
+        </Link>
+        <Link to={RouteUtils.getLatestBlocksRoute(Layer.Emerald)}>
```

Comment:
Was reviewed so not looking at code, but here we should have `Sapphire`

Reply:
Yes, I was planning to fix/replace all the links in the subsequent PRs, when we actually build those pages.

Reply:
Nice catch